assets/javascripts/discourse/controllers/create-channel-modal.js

```diff
@@ -33,6 +33,20 @@ export default Controller.extend(ModalFunctionality, {
     return !this.entitySelected || !name?.length > 0;
   },
 
+  @discourseComputed("category")
+  categoryHint(category) {
+    if (category) {
+      return {
+        link: `/c/${category.slug}/edit/security`,
+        category: category.name,
```

Comment:
I can't remember but are category names escaped by default? Otherwise, this may result in an XSS since we use it in a translation that is marked `html-safe`. Maybe we should escape this just to be safe?

Reply:
You're absolutely right